package.json

```diff
@@ -57,6 +58,7 @@
     "tslint-config-airbnb": "^5.11.0",
     "tslint-config-semistandard": "^7.0.0",
     "tslint-react": "^3.6.0",
+    "typed-css-modules-loader": "0.0.17",
```

Comment:
Isn't this also a webpack loader? Doesn't that mean we have two now?

Reply:
This brings the command line tool for typed css module loader.

Reply:
Isn't https://github.com/Quramy/typed-css-modules the main module for the cli tool?